ZenPacks/zenoss/LinuxMonitor/configure.zcml

```diff
@@ -5,6 +5,33 @@
     xmlns:zcml="http://namespaces.zope.org/zcml"
     >
 
+    <adapter
+        factory="ZenPacks.zenoss.LinuxMonitor.info.FileSystemInfo"
+        for=".FileSystem.FileSystem"
+        provides="ZenPacks.zenoss.LinuxMonitor.interfaces.IFileSystemInfo"
+        />
+
+    <adapter
+        factory="ZenPacks.zenoss.LinuxMonitor.info.HardDiskInfo"
+        for=".HardDisk.HardDisk"
+        provides="ZenPacks.zenoss.LinuxMonitor.interfaces.IHardDiskInfo"
+        />
+
+<!-- DynamicView -->
+    <configure zcml:condition="installed ZenPacks.zenoss.DynamicView">
+        <subscriber
+            provides="ZenPacks.zenoss.DynamicView.interfaces.IRelationsProvider"
+            for=".FileSystem.FileSystem"
+            factory=".dynamicview.FileSystemRelationsProvider"
+            />
+        <subscriber
+            provides="ZenPacks.zenoss.DynamicView.interfaces.IRelationsProvider"
+            for=".HardDisk.HardDisk"
+            factory=".dynamicview.HardDiskRelationsProvider"
+            />
+
+    </configure>
+
     <!-- Impact -->
     <configure zcml:condition="installed ZenPacks.zenoss.Impact">
```

Comment:
You shouldn't need this impact section in configure.zcml or impact.py now that you've implemented DynamicView IRelationsProvider adapters for FileSystem and HardDisk.